src/commands/utility/search-my-points.js

```diff
@@ -60,12 +60,17 @@ module.exports = {
       ? channels.map((channel) => `in:${channel}`).join(' ')
       : '';
 
+    const openedPRs = `before: ${endDateStr} after: ${startDateStr} ${channelQueryParts} Author: @${escapedUserId}`;
```

Comment:
I think this '@' is not needed
```suggestion
    const openedPRs = `before: ${endDateStr} after: ${startDateStr} ${channelQueryParts} Author: ${escapedUserId}`;
```
![image](https://github.com/user-attachments/assets/65218e4e-4d8d-42ba-88fb-03e0199e0d8c)

Reply:
with '@'
![image](https://github.com/user-attachments/assets/eb543fa6-45d2-49ee-9e10-9d4b0463af31)

without '@'
![image](https://github.com/user-attachments/assets/2abc57dc-b682-434a-a4c3-34aae9a0b23d)